### PR TITLE
Release 1.10.7

### PR DESCRIPTION
**Pull request by plemarquand**

### CODE REVIEW

**Review by award999 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 1.10.7 - 2024-08-08
+
+### Changed
+
+- Support the new unified testing binary format released with Xcode 15 Beta 5
```
Comment (award999) [inline CHANGELOG.md:7]: actually wait, is 16?

Comment (adam-fowler) [reply CHANGELOG.md:7]: It'll be 16. Is this format only used on macOS?

Comment (plemarquand) [reply CHANGELOG.md:7]: @adam-fowler It is used on all platforms. On macOS we need to use the swiftpm-testing-helper binary but on windows/linux we can just invoke the unified binary directly.

**Review by award999 — APPROVED** [review]

**Review by adam-fowler — APPROVED** [review]

**Comment by award999** [comment]
Verified

**Pull request merged**